superset/views/base.py

```diff
@@ -153,7 +153,7 @@ def json_response(self, obj, status=200):
             status=status,
             mimetype='application/json')
 
-    def common_bootsrap_payload(self):
```

Comment:
This made me super confused, since I was `grep`ping for `bootstrap` and couldn't find this method!